src/devices/dresden_elektronik.ts

```diff
@@ -32,6 +32,13 @@ const definitions: Definition[] = [
         extend: extend.light_onoff_brightness(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['Hive'],
+        model: 'Hive',
+        vendor: 'Phoscon',
+        description: 'Battery powered smart LED light',
+        extend: extend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device